lms/djangoapps/edraak_university/views.py

```diff
@@ -85,13 +110,33 @@ def get_success_url(self):
 
     def get_context_data(self, **kwargs):
         data = super(UniversityIDView, self).get_context_data(**kwargs)
+        university_id =get_university_id(self.request.user, self.kwargs['course_id'])
+
+        if university_id:
+            terms_and_conditions_read = university_id.terms_and_conditions_read
+        else:
+            terms_and_conditions_read = False
         data.update({
             'form': self.get_form(),
             'has_valid_information': has_valid_university_id(self.request.user, self.kwargs['course_id']),
+            'form_is_disabled': self.form_disabled(self.get_course().id),
+            'terms_conditions': self.get_university_id_settings(self.get_course().id).get_terms_and_conditions(),
+            'terms_and_conditions_read': terms_and_conditions_read ,
```

Comment:
~~Ok, this is a bit of twisted logic, the way you are doing it is you are calling the DB, getting a database object, where you call the get_terms_and_conditions method on, which calls helpers to check the language, which accordingly will send back the correct version of terms_and_conditions. Do you see it?
I believe a better way would be use the translations `get_language()` method which if arabic, then just access the saved arabic version of terms_and_conditions on the object, and no need for the whole hassle.~~ 
Anyway, this is not needed as we only need an arabic version of terms_and_conditions, check that with Lina.